[2024-08-12T01:22:48.521+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-12T01:22:48.533+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-12T01:22:47.329990+00:00 [queued]>
[2024-08-12T01:22:48.541+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-12T01:22:47.329990+00:00 [queued]>
[2024-08-12T01:22:48.542+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-12T01:22:48.547+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): weather_data_migration> on 2024-08-12 01:22:47.329990+00:00
[2024-08-12T01:22:48.551+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=156) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-12T01:22:48.552+0000] {standard_task_runner.py:64} INFO - Started process 158 to run task
[2024-08-12T01:22:48.552+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_migration', 'weather_data_migration', 'manual__2024-08-12T01:22:47.329990+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_migration_dag.py', '--cfg-path', '/tmp/tmp3uvky0is']
[2024-08-12T01:22:48.554+0000] {standard_task_runner.py:91} INFO - Job 126: Subtask weather_data_migration
[2024-08-12T01:22:48.592+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_migration.weather_data_migration manual__2024-08-12T01:22:47.329990+00:00 [running]> on host 230d4773b38e
[2024-08-12T01:22:48.638+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_migration' AIRFLOW_CTX_TASK_ID='weather_data_migration' AIRFLOW_CTX_EXECUTION_DATE='2024-08-12T01:22:47.329990+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-12T01:22:47.329990+00:00'
[2024-08-12T01:22:48.642+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-12T01:22:48.643+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-12T01:22:48.644+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/weather_data_migration.py --connection postgresql+psycopg2://***:***@postgres/***']
[2024-08-12T01:22:48.655+0000] {subprocess.py:86} INFO - Output:
[2024-08-12T01:22:49.173+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-12T01:22:49.177+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-12T01:22:49.205+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_migration, task_id=weather_data_migration, run_id=manual__2024-08-12T01:22:47.329990+00:00, execution_date=20240812T012247, start_date=20240812T012248, end_date=20240812T012249
[2024-08-12T01:22:49.228+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-12T01:22:49.244+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-12T01:22:49.244+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
